cli/dcos-spark/security.go

```diff
@@ -0,0 +1,236 @@
+package main
+
+import (
+	"fmt"
+	"errors"
+	"crypto/rand"
+	"encoding/base64"
+	"strings"
+	"log"
+)
+
+
+const KEYLENGTH = 128
+const SECRET_REFERENCE_TEMPLATE = "spark.mesos.%s.secret.names"
```

Comment:
From what I can tell, there are only two options for `%s`. Instead of using `Sprintf` every time, we could use a map instead.